digitalpublicgoods/upyog.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "UPYOG",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://upyog.niua.org/citizen/language-selection"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://docs.digit.org/"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "checkNonPIIAccessMechanism": "",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Unknown",
+    "privacyComplianceList": [
+      ""
+    ],
+    "adherenceSteps": [
+      ""
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "https://niua.in/intranet/sites/default/files/2454.pdf"
+    ],
+    "evidenceStandardSupport": [
+      "https://niua.in/intranet/sites/default/files/2454.pdf"
+    ],
+    "implementBestPractices": "Unknown",
+    "bestPracticesList": [
```

Comment:
Please check with project which best practices were adhered to when designing the platform.